lib/galaxy/datatypes/microarrays.py

```diff
@@ -1,6 +1,7 @@
 import logging
```

Comment:
Need to add `import struct` here

Reply:
Done, I didn't want to surcharge sniffing with bytes unpacking. I will go with your suggestion.

Reply:
Thanks! You need to keep the empty line separating Standard Library imports from the others.